bin/haraka

```diff
@@ -223,6 +223,7 @@ function setupHostname (confPath) {
 function setupBaseConfig (confPath) {
     copyFile(path.join(base, 'config', 'smtp.ini'), path.join(confPath, 'smtp.ini'));
     copyFile(path.join(base, 'config', 'loglevel'), path.join(confPath, 'loglevel'));
+    copyFile(path.join(base, 'config', 'logformat'), path.join(confPath, 'logformat'));
```

Comment:
this added line can be removed and the line above it (loglevel) should be replaced with log.ini.